src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -150,7 +150,11 @@ export const AGGREGATIONS = {
     parameters: [],
     outputType: 'date',
     isSortable: true,
-    multiPlotType: 'area',
+  },
+  latest_event: {
+    parameters: [],
+    outputType: 'string',
+    isSortable: true,
```

Comment:
Does this field render as a link? We used to use the `latest_event()` function for building links to stacks of events, but with the newer drilldown flow we haven't been using `latest_event()` at all. If we don't have solid use cases for it, we should consider removing it, to help manage complexity and reduce the surface area of discover a tiny bit.

Reply:
👍 Didn't realize we weren't using it at all, I'll remove it then so its not a new discover column too